pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -2023,6 +2024,10 @@
       heading: "Create music with code"
       desc: "Get creative with your favorite artist's music and code new songs and mixes!"
     featuring_songs_by: "Featuring songs by:"
+    artist_list: "%{artist_list}, and more!"
```

Comment:
Will all languages work if we programmatically generate a comma-separated list?

Reply:
To Brendan's question: Are you using a programmatically inserted list here because we don't want the names translated? Or because we want to be able to easily change the list? Or some other reason?

Reply:
I initially did it this way because we don't want the names translated, but now that you mention it there is probably a high chance that the list will change.

Reply:
I'm okay moving forward with this. I think Brendan brings up a valid concern, but I'm not sure of the solution, other than spelling out all the names.

I think we do use names in Dance Party strings, though, right?

Reply:
I have seen a list of artist names in some Dance Party strings in Crowdin before. 
There are no specific rules on how to handle proper names. While it is true that [some languages](https://linguistics.stackexchange.com/questions/6737/what-languages-use-the-serial-comma) don't use comma-separated values. 
Kelby's approach allows for simultaneously updating the list for all languages.
It would be a question of preference: Do we prefer to acustome more to other languages? or Do we prefer to have an accurate list for all users?

Reply:
Who is the decider in this case? And I'll remove the Oxford comma, we don't have that on code.org/dance (which **does not** have artists name translations in a similar case where this will be used): https://github.com/code-dot-org/code-dot-org/blob/0c302bcd22c299f2136e32d0e8a23ff8ce5112ca/pegasus/sites.v3/code.org/views/dance/featured_artists.haml#L13)

Reply:
I think you're the decider Kelby ;) 

What I'm hearing Mario say is -- if we think that list might change, it's probably a good idea to go with your change so that the list is always accurate. If it's definitely not going to change, it could be good to make sure the string is properly formatted for all languages.

Reply:
Ooo how exciting haha! I'm going to go with what I have, Dani shared more artist info with me yesterday and I expect it to change. We can always revisit if needed.

Reply:
TIL [Intl.ListFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat).